api/lib/queries.js

```diff
@@ -127,7 +127,7 @@ exports.execUpdateSetManyToOne = async (
   }
   let updatedOneModel = await oneModel.findByIdAndUpdate(
     oneObj._id,
-    { $set: { [oneKeyOfMany]: manyObjs.map((fl) => fl._id) } },
+    { $addToSet: { [oneKeyOfMany]: manyObjs.map((fl) => fl._id) } },
```

Comment:
I don't think is right? Shouldn't this be "setting" the value rather than "adding"/appending it?

Reply:
When it was $set and I added two resources under the same user, it would replace the previous resource. Shouldn't it be addset to add to list of resources someone's uploaded?

Reply:
"execUpdateSetManyToOne" abstract says what things are rather than how they should be changed. This issue w/addSet is that it will never remove resources that were re-assigned to a different user. I think re-thinking this as setResources should fix this.

Reply:
Ok that makes sense, will change to setResources and back to set. thanks